app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -55,6 +55,10 @@ def stored_result
       @stored_result = document_capture_session&.load_result
     end
 
+    def confirm_selfie_performed_if_needed
+      !decorated_sp_session.selfie_required? || stored_result.selfie_check_performed
+    end
+
```

Comment:
@jmhooper Sonia and I tested this change and then also tested in `main`. We saw the expected behavior in both places (could not continue past the doc auth page without a selfie if the requirements were changed partway through). Do you have any advice on if there's something we're missing that's needed here? Or maybe these changes aren't needed?

(Putting the comment here so that the conversation can be chained / followed)

Reply:
Slack discussion [here](https://gsa-tts.slack.com/archives/CNCGEHG1G/p1704816646145549). The conclusion is that controller tests are sufficient for this.